### ISSUE TITLE
Add a `Reviewer Tools` item to the user dropdown on AMO

### ISSUE THREAD

**Issue by jvillalobos** [mozilla/addons-server#7280]
Part of #7172

Add a Reviewer Tools link to the user dropdown on AMO (under Tools), pointing to https://reviewers.addons.mozilla.org/reviewers/

Anyone with any of the following permissions should have access to that page and see the link: 
`Addons:PostReview`, `Addons:ContentReview`, `Addons:Review`, `Ratings:Moderate`, `Personas:Review`, `Addons:ReviewUnlisted`, `Addons:Edit`.

**Comment by AlexandraMoga** [comment]
Verified the contents of the user menu with users belonging to the following groups: `Addons:PostReview,` `Addons:ContentReview`, `Addons:Review`, `Ratings:Moderate`, `Personas:Review`,  `Admin`, `Addons:ReviewUnlisted`
All these users have the 'Reviewer Tools' option included in the drop down menu.

![image](https://user-images.githubusercontent.com/31961530/35154178-e716dbe0-fd31-11e7-8bef-3bb997d5c3b4.png)


@diox, I have a question regarding 'Staff' users: weren't they expected to have 'read only' permission? I was able to approve/reject add-ons/content with a user included in this category. This is also the only group to include the `Addons:edit` permission, as far as I've seen.
